apiary.apib

```diff
@@ -3047,6 +3050,33 @@ owner or have edit permissions.
 
 + Response 204
 
+## Add default subject link [/collections/{id}/links/default_subject]
+Links a default subject to a collection. This subject's first media
+location URL will be included in the serialized collection and used
+as the thumbnail. Update this attribute with `null` to use the first
```

Comment:
This? https://github.com/zwolf/Panoptes/blob/6a1f7806ad2c5bf2e4a0de2b8c98d9ae7ec98d56/app/serializers/collection_serializer.rb#L26